src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  max-width: 944px;
+  padding: 50px 0;
+
+  display: flex;
+  align-items: flex-start;
+  flex-wrap: wrap;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 47px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }

```

Comment:
You can use 
    $__card:not(:nth-child(4n)) {
       margin-right: 48px;
    }